indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -438,20 +454,70 @@ function EditableListDisplay({
       />
     );
   };
+  const renderTags = (__, editable) => {
+    const tags = editable.tags.sort((a, b) => a.code.localeCompare(b.code));
+    const labelTags = () => {
+      return (
+        <div style={{display: 'flex', flexWrap: 'nowrap'}}>
+          {tags.map(t => {
+            return (
+              <Label key={t.code} size="small" color={t.color}>
+                {t.code}
+              </Label>
+            );
+          })}
+        </div>
+      );
+    };
+
+    const colorTags = () => {
+      return (
+        <div>
+          {tags.map(t => {
+            return <Label key={`${t.code}-dot`} circular empty color={t.color} />;
+          })}
+        </div>
+      );
+    };
+
+    return [labelTags(), colorTags()];
+  };
   const renderFuncs = {
     friendlyId: renderId,
     title: renderTitle,
     code: renderCode,
     revision: editable => (editable ? editable.revisionCount : ''),
     status: renderStatus,
     editor: renderEditor,
+    tags: renderTags,
   };
 
   // eslint-disable-next-line react/prop-types
   const renderCell = ({dataKey, rowIndex}) => {
     const row = sortedList[rowIndex];
     const fn = renderFuncs[dataKey];
-    return (
+
+    return dataKey === 'tags' && row.editable?.tags ? (
+      <Popup
+        position="top center"
+        content={fn(row[dataKey], row.editable, rowIndex)[0]}
+        trigger={
+          row.editable.tags.length > 1 ? (
+            <div styleName="rowcolumn-tooltip" role="gridcell">
+              {fn(row[dataKey], row.editable, rowIndex)[1]}
+              {row.editable.tags.length > 5 ? (
+                <div className="more-indicator">
+                  <Icon name="plus circle" color="grey" />
+                </div>
+              ) : null}
+            </div>
+          ) : (
+            <div>{fn(row[dataKey], row.editable, rowIndex)[0]}</div>
```

Comment:
Just to avoid some repetition (and maybe I'd go for an object with keys rather than an array, it's easy to confuse the indexes):

```suggestion
    if (dataKey === 'tags' && row.editable?.tags) {
      const [labels, colors] = fn(row[dataKey], row.editable, rowIndex)
      return (
        <Popup
          position="top center"
          content={labels}
          trigger={
            row.editable.tags.length > 1 ? (
              <div styleName="rowcolumn-tooltip" role="gridcell">
                {colors}
                {row.editable.tags.length > 5 ? (
                  <div className="more-indicator">
                    <Icon name="plus circle" color="grey" />
                  </div>
                ) : null}
              </div>
            ) : (
              <div>{labels}</div>
            )
          }
          on="hover"
        />
      );
    }
```